.travis.yml

```diff
@@ -87,12 +92,12 @@ before_install:
     - sudo apt-get -qq update
     - sudo apt-get install -y mongodb-org
 
-    # Install swagger-cli
-    - sudo apt-get -qq update && sudo apt-get -qq install -y npm
+
+    - sudo apt-get -qq update && sudo apt-get -qq install -y npm e2tools
```

Comment:
It is, but do I need to add even more complexity to the travis file by adding more conditional statements?